cfgov/jinja2/v1/_includes/molecules/info-unit.html

```diff
@@ -0,0 +1,79 @@
+{# ==========================================================================
+
+   Info Unit
+
+   ==========================================================================
+
+   Description:
+
+   Create an info-unit molecule when given:
+
+   value.modifier:  A modifier class used to alternate the info unit from
+                    stacked to inline layouts
+
+   value.image:     An object containing:
+
+                    url:           The image url
+                    alt:           Image alt
+                    is_square:     Whether the image is square or not.
+                                   Default is false
+                    is_decorative: Whether the image is a bg image or not.
+                                   Default is false
+
+   value.heading:   A string containing title of module.
+
+   value.body:      A string containing the content to display below the
+                    heading (typically a paragraph).
+
+   value.is_button: Boolean if the link should be a button.
+                    Defaut is false
+
+   value.links:     A tuple of items to create a list of links. Objects include
+                    a url and text to display.
+
+                    link_url:  A string for the URL of the link.
+                    link_text: A string for the text of the link.
+
+
+   ========================================================================== #}
+
+{% macro info_unit(value) %}
```

Comment:
Currently the code base is mixed, but I agree we should decide on a single pattern. While building this, I realized I'm not so sure `render` is the correct choice because it requires you import `as`, which can lead to naming discrepancies. 

```
{% import 'molecules/info_unit.html' as anything_goes with context %}
{% anything_goes.render( value ) %}
```